### PR TITLE
Version bump, polish, crate name update

### PR DESCRIPTION
**Pull request by kvark**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [package]
-name = "metal-rs"
-version = "0.10.4"
+name = "metal"
```
Comment (grovesNL) [inline Cargo.toml]: 🎉

File: src/device.rs [hunk 2]
```diff
@@ -61,8 +59,9 @@ bitflags! {
     }
 }
 
+#[allow(non_camel_case_types)]
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
-enum OS {
+pub enum OS {
```
Comment (grovesNL) [inline src/device.rs]: I thought this was [purposely kept private](https://github.com/gfx-rs/metal-rs/pull/62#discussion_r206106515) for now?

Comment (kvark) [reply src/device.rs]: good point!

**Comment by kvark** [comment]
fixed now
bors r=grovesNL

**Pull request merged**